lib/rubocop/cop/utils/regexp_ranges.rb

```diff
@@ -14,7 +14,7 @@ def initialize(root)
         end
 
         def pairs
-          unless @pairs
+          unless defined?(@pairs)
```

Comment:
Can you initialize `@pairs` with the `initialize` method instead of using `define?`?

```diff
diff --git a/lib/rubocop/cop/utils/regexp_ranges.rb b/lib/rubocop/cop/utils/regexp_ranges.rb
index aa60666c9..6322091c5 100644
--- a/lib/rubocop/cop/utils/regexp_ranges.rb
+++ b/lib/rubocop/cop/utils/regexp_ranges.rb
@@ -11,6 +11,7 @@ module RuboCop
         def initialize(root)
           @root = root
           @compound_token = []
+          @pairs = nil
         end

         def pairs
```